pkg/model/resources/nodeup.go

```diff
@@ -53,12 +53,9 @@ NODEUP_HASH={{ NodeUpSourceHash }}
 {{ ProxyEnv }}
 
 function ensure-install-dir() {
-  INSTALL_DIR="/var/cache/kubernetes-install"
-  # On ContainerOS, we install to /var/lib/toolbox install (because of noexec)
```

Comment:
I _think_ we're going to have to keep the COS workaround, because at this point /opt/kops isn't writeable.  I think we should make it writeable (and I started that in https://github.com/kubernetes/kops/pull/8269), but I don't particularly want to do so in bash.

That said, I'm also happy to break this briefly and then fix it in #8269...

Reply:
I took a look and these are our options that I believe are mounted rw & exec at boot-time.  /var/lib/toolbox seems no worse than our other choices...

```
/dev/sda1 on /var/lib/google type ext4 (rw,nosuid,nodev,relatime,commit=30)
/dev/sda1 on /var/lib/docker type ext4 (rw,nosuid,nodev,relatime,commit=30)
/dev/sda1 on /var/lib/toolbox type ext4 (rw,nodev,relatime,commit=30)
```